docker/generate-maps

```diff
@@ -1,80 +0,0 @@
-#!/bin/bash
```

Comment:
I believe that everything previous handled by this script is now handled by the sum of the `cibuild` scripts in the repository.